code/java/search1.java.out

```diff
@@ -0,0 +1,2 @@
+1: #001 Ansonia Sunwatch (pocket compas dial)
```

Comment:
I assume that this would fail at the moment because the docid is formatted as `001` but the code will output it without the 0-padding.